Formula/loris.rb

```diff
@@ -0,0 +1,27 @@
+class Loris < Formula
+  desc "Audio analysis, manipulation and synthesis"
+  homepage "http://www.cerlsoundgroup.org/Loris"
+  url "https://downloads.sourceforge.net/code-snapshots/git/l/lo/loris/loris-code.git/loris-loris-code-27d14aef9e50573bccb3e11d9c6b5c051ea04862.zip"
```

Comment:
Is it still under active development? If not, and the last stable release is from 2010, then we won't be inclined to accept it.